frontend/packages/console-app/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
+    "@console/metal3": "0.0.0-fixed",
```

Comment:
Committing this change to git means that Metal 3 plugin will be included in development and production builds by default.

Until the Metal 3 plugin leaves the experimental phase, I'd suggest to treat this as a local change and not commit to git.

Reply:
Done